### PR TITLE
done

### PR DESCRIPTION
**Pull request by asya-bm**
- [DEMO LINK](https://asya-bm.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/watch.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+@keyframes rotate-second {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```
Comment (Viktor-Kost) [inline src/styles/watch.scss:54]: there are the same animations, so no needs to use two different ones

File: src/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+@mixin hand {
+  position: absolute;
+  background-color: #000;
+  left: 50%;
+  transform-origin: bottom center;
+  width: $hand-width;
+  border-top-right-radius: 90%;
+  border-top-left-radius: 90%;
+  border-bottom-right-radius: 90%;
+  border-bottom-left-radius: 90%;
```
Comment (Viktor-Kost) [inline src/styles/variables.scss:15]:
```suggestion
  border-radius: 90%;
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
GJ!
Some minor fixes needed

**Review by demetrxx — APPROVED** [review]
Well done 👍 Go ahead!

**Pull request closed without merging**